sharktank_models/README.md

```diff
@@ -97,3 +97,35 @@ built as part of the [shark-ai project](https://github.com/nod-ai/shark-ai).
 
     See also
     https://docs.pytest.org/en/latest/how-to/output.html#creating-junitxml-format-files
+
+## Running quality tests
+
+Please refer to [Quality tests README](regression_tests/README.md) to run tests
+
+## Running benchmark tests
+
+Please refer to [Benchmark tests README](benchmarks/README.md) to run tests
+
+Note: for benchmark tests to run, you will need `vmfbs` files available
+
+## Generating model files using Shark AI
+
+In order to generate and compile MLIR files to compile, run quality tests and benchmarking tests, please run the following the following commands:
+
+This example generates IRPA and MLIR files for Llama, please look in [Shark AI Models](https://github.com/nod-ai/shark-ai/tree/main/sharktank/sharktank/models) to see which models you can generate
+
+```
+git clone https://github.com/nod-ai/shark-ai.git
+
+cd shark-ai/sharktank
+python3 -m pip install .
```

Comment:
I see that these instructions are forked from `llama3.1/README.md`. They are fine as-is, but we should generally steer users and test suites towards package builds instead of source builds. That would mean replacing

```
git clone https://github.com/nod-ai/shark-ai.git
cd shark-ai/sharktank
python3 -m pip install .
```

with

```
python3 -m pip install sharktank
```

for stable releases or this for nightly releases, following https://github.com/nod-ai/shark-ai/blob/main/docs/nightly_releases.md:

```
python3 -m pip install sharktank -f https://github.com/nod-ai/shark-ai/releases/expanded_assets/dev-wheels --pre
```